### PR TITLE
OHARA-1670 Implement a simple cache to speed up ssh collie

### PR DESCRIPTION
**Pull request by chia7712**

### CODE REVIEW

**Comment by jackyoh** [comment]
retry it

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -42,6 +42,7 @@ versions["commonNet"] = project.hasProperty('commonNet.version') ? project.prope
 versions["sshd"] = project.hasProperty('sshd.version') ? project.properties['sshd.version'] : "2.0.0"
 // commonIo has included by apache hadoop so we use the identical version
 versions["commonsIo"] = project.hasProperty('commonsIo.version') ? project.properties['commonsIo.version'] : "2.4"
+versions["commonsIo"] = project.hasProperty('commonsIo.version') ? project.properties['commonsIo.version'] : "2.4"
```
[the remark this answers is not in the document]
Comment (chia7712) [reply gradle/dependencies.gradle]: 感謝

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
I have no enough cycle to complete this PR. Will move on up this PR in 0.3

**Pull request closed without merging**